src/app/content/taruni-gopi/taruni-gopi.component.html

```diff
@@ -9,12 +9,12 @@
   <!-- Botões da página inicial -->
 
   <div class="homepage-buttons-box">
-    <button class="homepage-buttons-box__button" (click)="toggle.createRegister = true; toggle.registerConfirmation = false">Register new devotee</button>
-    <button class="homepage-buttons-box__button" (click)="toggle.readRegister = true; toggle.registerConfirmation = false">Show registers</button>
+    <button class="homepage-buttons-box__button" (click)="toggle.createRegister.show(); toggle.registerConfirmation.hide()">Register new devotee</button>
+    <button class="homepage-buttons-box__button" (click)="toggle.readRegister.show(); toggle.registerConfirmation.hide()">Show registers</button>
```

Comment:
Eu prefiro que voce abstraia os comandos do click dentro de uma outra funcao.  Ela pode se chamar por exemplo handleRegisterButton() { this.toggle.createRegister.show(); toggle.registerConfirmation.hide() }
 E ai foce chama essa nova funcao no click. Fica feio click componto e virgula